src/logic/map_objects/tribes/worker_program.cc

```diff
@@ -443,7 +455,7 @@ findspace
 /**
  * iparam1 = radius
  * iparam2 = FindNodeSize::sizeXXX
- * iparam3 = whether the "space" flag is set
+ * iparam3 = 1st bit: whether the "space" flag is set; 2nd bit: whether the no_notify flag is set.
```

Comment:
Should we add a TODO her to put all flags into one Iparam?
Or use a new Iparam for the new flag. 
otherwise the flag design is not consistent in this function